### PR TITLE
Address grid component comments.

### PR DESCRIPTION
**Pull request by michelle192837**
Followup to comments on #1192 . (Some already addressed by #1207).

For row-id vs. row-name, I realized that given how the component renders, it's less of an ID and more of a display name thing, so I think 'name' feels more accurate? Happy to roll that part back though.

Storybook: 8noJSJdFQSab6EU
Local: 
54vtgxKFZUwx2Dk

### CODE REVIEW

File: web/src/testgrid-grid-header-row.ts [hunk 1]
```diff
@@ -1,8 +1,8 @@
-import { LitElement, html, css } from "lit";
-import { map } from "lit/directives/map.js";
-import { customElement, property } from "lit/decorators.js";
+import { LitElement, html, css } from 'lit';
+import { map } from 'lit/directives/map.js';
+import { customElement, property } from 'lit/decorators.js';
 import { ListHeadersResponse } from './gen/pb/api/v1/data.js';
-import './testgrid-grid-row-id';
+import './testgrid-grid-row-name.js';
```
Comment (sultan-duisenbay) [inline web/src/testgrid-grid-header-row.ts:5]: nit: .js is not necessary

File: web/src/testgrid-grid-row.ts [hunk 2]
```diff
@@ -1,9 +1,9 @@
-import { LitElement, html, css } from "lit";
-import { map } from "lit/directives/map.js";
-import { customElement, property } from "lit/decorators.js";
+import { LitElement, html, css } from 'lit';
+import { map } from 'lit/directives/map.js';
+import { customElement, property } from 'lit/decorators.js';
 import { ListRowsResponse_Row } from './gen/pb/api/v1/data.js';
 import { TestStatus } from './gen/pb/test_status/test_status';
-import './testgrid-grid-row-id';
+import './testgrid-grid-row-name.js';
```
Comment (sultan-duisenbay) [inline web/src/testgrid-grid-row.ts:6]: nit: ditto

File: web/stories/testgrid-grid-row-header.stories.ts [hunk 3]
```diff
@@ -1,9 +1,9 @@
 import { html, TemplateResult } from 'lit';
```
Comment (sultan-duisenbay) [inline web/stories/testgrid-grid-row-header.stories.ts:1]: nit: I guess the story name has to be changed to `xx-row-name`?

File: web/test/testgrid-grid-row-name.test.ts [hunk 4]
```diff
@@ -0,0 +1,26 @@
+import {
+  html,
+  fixture,
+  defineCE,
+  unsafeStatic,
+  expect,
+} from '@open-wc/testing';
+import { TestgridGridRowName } from '../src/testgrid-grid-row-name';
+
+describe('TestGrid grid row header', () => {
```
Comment (sultan-duisenbay) [inline web/test/testgrid-grid-row-name.test.ts:10]: nit: row name

**Review by sultan-duisenbay — APPROVED** [review]
/hold

LGTM with a couple of nits. Thanks!

**Comment by michelle192837** [comment]
Thanks! Changes made, and verified the Storybook stories and local run still work ^^

**Pull request merged**